fapi-client/src/main/scala/com/gu/facia/api/FAPI.scala

```diff
@@ -153,27 +152,6 @@ object FAPI {
     yield Collection.draftContent(collection, setOfContent, snapContent)}
 
   /**
-   * Fetches content for the given backfill query. The query can be manipulated for different
-   * requirements by providing adjustment functions. The results then have their facia metadata
-   * resolved using the collection information.
-   */
-  @deprecated
-  def backfill(backfillQuery: String, collection: Collection,
```

Comment:
`backfillQuery` should not be exposed to users of the library. This method is made obsolete by  `backfillFromConfig`